bootstrap.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+#######################################################################################################################
+# Crate Universe
+#######################################################################################################################
+
+# Find the location of the script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+BOOTSTRAP_DIR="${SCRIPT_DIR}/crate_universe/private/bootstrap"
+
+# Go to root of rules_rust
+pushd "${SCRIPT_DIR}" &> /dev/null
+has_changes="$(git log origin/main.. -- crate_universe/**)"
+popd &> /dev/null
+
+# Only bootstrap the binary if there are changes
+if [[ -n "${has_changes}" ]]; then
+    pushd "${BOOTSTRAP_DIR}" &> /dev/null
+    bazel run //:build && bazel run //:install && bazel clean
```

Comment:
Why the `bazel clean`?

Reply:
So we avoid recursive symlinks when using `override_repository`. The clean should be fast since the only thing built by bazel is a shell script.